.golangci.yml

```diff
@@ -17,7 +17,7 @@ linters:
     - containedctx
     - contextcheck
     - decorder
-    - depguard
+    # - depguard
```

Comment:
Depguard now fails on all non-stdlib dependencies by default. If we want to configure this, we will want to tweak it separately in each downstream project. For now, I've disabled the linter.